manifests/plugin/abrt.pp

```diff
@@ -5,67 +5,47 @@
 #
 # $spooldir::                   Directory where uReports are stored before they are sent
-#                               type:Stdlib::Absolutepath
 #
 # $aggregate_reports::          Merge duplicate reports before sending
-#                               type:Boolean
 #
 # $send_period::                Period (in seconds) after which collected reports are forwarded.
 #                               Meaningful only if smart-proxy-abrt-send is run as a daemon (not from cron).
-#                               type:Integer[0]
 #
 # $faf_server_url::             FAF server instance the reports will be forwarded to
-#                               type:Optional[String]
 #
 # $faf_server_ssl_noverify::    Set to true if FAF server uses self-signed certificate
-#                               type:Boolean
 #
 # $faf_server_ssl_cert::        Enable client authentication to FAF server: set ssl certificate
-#                               type:Optional[Stdlib::Absolutepath]
 #
 # $faf_server_ssl_key::         Enable client authentication to FAF server: set ssl key
-#                               type:Optional[Stdlib::Absolutepath]
 #
 # === Advanced parameters:
 #
 # $enabled::                    Enables/disables the abrt plugin
-#                               type:Boolean
 #
 # $group::                      group owner of the configuration file
-#                               type:Optional[String]
 #
 # $listen_on::                  Proxy feature listens on http, https, or both
-#                               type:Foreman_proxy::ListenOn
 #
 # $version::                    plugin package version, it's passed to ensure parameter of package resource
 #                               can be set to specific version number, 'latest', 'present' etc.
-#                               type:Optional[String]
 #
 class foreman_proxy::plugin::abrt (
-  $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $listen_on               = $::foreman_proxy::plugin::abrt::params::listen_on,
-  $version                 = $::foreman_proxy::plugin::abrt::params::version,
-  $group                   = $::foreman_proxy::plugin::abrt::params::group,
-  $abrt_send_log_file      = $::foreman_proxy::plugin::abrt::params::abrt_send_log_file,
-  $spooldir                = $::foreman_proxy::plugin::abrt::params::spooldir,
-  $aggregate_reports       = $::foreman_proxy::plugin::abrt::params::aggregate_reports,
-  $send_period             = $::foreman_proxy::plugin::abrt::params::send_period,
-  $faf_server_url          = $::foreman_proxy::plugin::abrt::params::faf_server_url,
-  $faf_server_ssl_noverify = $::foreman_proxy::plugin::abrt::params::faf_server_ssl_noverify,
-  $faf_server_ssl_cert     = $::foreman_proxy::plugin::abrt::params::faf_server_ssl_cert,
-  $faf_server_ssl_key      = $::foreman_proxy::plugin::abrt::params::faf_server_ssl_key,
+  Boolean $enabled = $::foreman_proxy::plugin::abrt::params::enabled,
```

Comment:
I'm not sure it's worthwhile maintaining the indentation with the length of some data types, and needing to re-align when changing or extending data types, but I can do I suppose. (Though if it's later migrated to module data, then little or no alignment is necessary any more either.)